ThirdParty.json

```diff
@@ -0,0 +1,107 @@
+[
+  {
+    "name": "Async++",
+    "url": "https://github.com/Amanieu/asyncplusplus",
+    "license": "MIT"
+  },
+  {
+    "name": "Catch2",
+    "url": "https://github.com/catchorg/Catch2",
+    "license": "Boost Software License 1.0"
+  },
+  {
+    "name": "cesium-native",
+    "url": "https://github.com/CesiumGS/cesium-native",
+    "license": "Apache 2.0"
+  },
+  {
+    "name": "cmake-modules",
+    "url": "https://github.com/bilke/cmake-modules",
+    "license": "Boost Software License 1.0"
+  },
+  {
+    "name": "cpp-httplib",
+    "url": "https://github.com/yhirose/cpp-httplib",
+    "license": "MIT"
+  },
+  {
+    "name": "CSPRNG",
+    "url": "https://github.com/Duthomhas/CSPRNG",
+    "license": "Boost Software License 1.0"
+  },
+  {
+    "name": "Draco",
+    "url": "https://github.com/google/draco",
+    "license": "Apache 2.0"
+  },
+  {
+    "name": "earcut",
+    "url": "https://github.com/mapbox/earcut.hpp",
+    "license": "ISC License"
+  },
+  {
+    "name": "Font Awesome Icons",
```

Comment:
@baothientran We don't include Font Awesome in Cesium Native

Reply:
Updated